src/IpAddress/ScoreConnection.qml

```diff
@@ -1,7 +1,7 @@
 /*
   * Connection Button :
   * - at the top left of the interface with others buttons
-  * - visible when the remote is disconnected form score
```

Comment:
no this is correct

(however I guess afterwards you mean "from score", not "form score")